packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -21,91 +23,51 @@ import { ProgressRadial } from '../ProgressIndicators/index.js';
 
 const TileContext = React.createContext<
   | {
+      status: 'positive' | 'warning' | 'negative' | undefined;
```

Comment:
nitpicky, but you can change this to `status?` and removed the `undefined` option

Reply:
Updated!